activejob/lib/active_job/test_helper.rb

```diff
@@ -736,7 +736,7 @@ def prepare_args_for_assertion(args)
           end
 
           if arguments[:at].acts_like?(:time)
-            at_range = arguments[:at] - 1..arguments[:at] + 1
+            at_range = arguments[:at] - 1.second..arguments[:at] + 1.second
```

Comment:
Since there's no `assert_not_enqueued_with` helper, I wasn't sure the best way to test this. One alternative would be to bypass the `assert_enqueued_with` altogether, and manually look at the value set.

Reply:
It would be valuable to continue to explore adding test coverage. 

Either way, a change like this warrants adding an entry to [activejob/CHANGELOG.md](https://github.com/rails/rails/blob/main/activejob/CHANGELOG.md).